reproducibility_tests/README.md

```diff
@@ -72,7 +79,7 @@ Reprodicibility tests are performed at the end of `examples/hybrid/driver.jl`, a
  0) Run a simulation, with a particular `job_id`, to the final time.
```

Comment:
Is there anything that needs to be added to this `Developer guide to reproducibility tests` due to the changes in this PR?

Reply:
No, the user interaction is effectively the same